docs/e2e-test.md

```diff
@@ -26,3 +26,31 @@ After a finished test, cleanup is performed automatically. If the test setup fai
 ```sh
 make clean -C /opt/metal3-dev-env/metal3-dev-env/ && sudo rm -rf /opt/metal3-dev-env/
 ```
+
+## Included tests
+
+The e2e tests currently run one after the other, using the same environment for all of them.
+They run in this order:
+
+1. Remediation
+1. Inspection *
+1. Pivoting
+1. Upgrade BMO
+1. Upgrade Ironic
+1. Certificate rotation
+1. Node reuse
+1. Re-pivoting
+
+*) Inspection is actually run [in the middle of the remediation test](https://github.com/metal3-io/cluster-api-provider-metal3/blob/8d08f375de93a793f839b42b5ec40e6bebf98664/test/e2e/remediation_test.go#L108) for practical reasons at the moment.
```

Comment:
This part with * looks a bit weird when viewed this file with "Display the rich diff"
![image](https://user-images.githubusercontent.com/40443040/157208740-565a8546-7c2b-4d7a-aa6d-d1503240d3d6.png)

Reply:
Not sure I understand. I added a `*` after inspection since it is a bit special that it runs in the middle of the remediation test, and added the explanation here. Any suggestion how to make it clearer?

Reply:
@lentzi90 yes, exactly that is what I meant, sorry was not clear, how about instead of **¹)**  just **¹** ? i.e:

¹Explanation here

Reply:
Updated!